tests/e2e/local/common_linux.sh

```diff
@@ -73,3 +73,23 @@ function install_kubectl() {
     sudo mv ./kubectl /usr/local/bin/kubectl
   fi
 }
+
+function check_and_install_packages() {
```

Comment:
Probably add some comments for the expected input args

Reply:
Done. PTAL, Thanks!